Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml

```diff
@@ -577,6 +577,57 @@
       - id: PlushieNuke
         orGroup: GiftPool
 
+- type: entity
+  parent: HappyHonkNukie
+  id: HappyHonkNukieSnacks
+  suffix: Toy Unsafe, Snacks
+  name: syndicate snack box
+  components:
+  - type: Item
+    size: 64
+  - type: Storage
+    capacity: 64 # need more room for goodies
+  - type: StorageFill
+    contents:
+    # toy
+    - id: C4
```

Comment:
Keep in mind this can show up on light syndicate crates.

Reply:
im wanting to tentatively allow more explosives anyway since we are slowly moving to a stricter requirement (whitelist, time spent etc.) before we let players do PvP like things anyway